tests/models/sam/test_modeling_sam.py

```diff
@@ -751,3 +751,9 @@ def test_inference_mask_generation_three_boxes_point_batch(self):
         iou_scores = outputs.iou_scores.cpu()
         self.assertTrue(iou_scores.shape == (1, 3, 3))
         torch.testing.assert_allclose(iou_scores, EXPECTED_IOU, atol=1e-4, rtol=1e-4)
+
+    def test_dummy_pipeline_generation(self):
+        generator = pipeline("mask-generation", model="facebook/sam-vit-base", device=torch_device)
+        raw_image = prepare_image()
+
+        _ = generator(raw_image, points_per_batch=64)
```

Comment:
I am surprised that this is not decorated by `require_vision`, but even more surprised that this test file has no `require_vision` at all, but having 

```python
if is_vision_available():
    from PIL import Image
```

This has to be fixed, but it's OK if we do this in a separate PR and maybe delegate the task to the original SAM modeling author in `transformers`.